crosswalks/usgs-modelcatalog.csv

```diff
@@ -0,0 +1,74 @@
+Property,usgs-modelcatalog
+codeRepository,software.usgs_software_gitlab_release_urls / software.external_software_repository_urls
+programmingLanguage,programming_language
+review,
+runtimePlatform,
+targetProduct,
+applicationCategory,item_type
+applicationSubCategory,type_keywords
+downloadUrl,software.download_url
+fileSize,
+installUrl,
+memoryRequirements,
+operatingSystem,
+permissions,
+processorRequirements,
+releaseNotes,
+softwareHelp,
+softwareRequirements,
+softwareVersion,version
+storageRequirements,
+supportingData,
+author,person.name when person_type=Author
```

Comment:
```suggestion
author,person when person_type=Author
```

Codemeta's `author`, like usgs-modelcatalog's `person`,  points to an object; and then we map that object's properties individually. `name` is already mapped below.